### ISSUE TITLE
C# backend doc/parameter inconsistency

### ISSUE THREAD

**Issue by jadamcrain** [stepfunc/oo_bindgen#27]
```
 /**
* initialize a WriteResult to indicate a successful write operation
* @param raw_exception Raw Exception code to include in the result
* @return WriteResult initialized to indicate a successful write operation
*/
public static WriteResult createRawException(UByte rawException)
```

parameter is properly camel-cased but doc paramter isn't

**Comment by emgre** [comment]
This will be fixed in #14.

**Comment by jadamcrain** [comment]
Fixed in #29
